tests/test_data.py

```diff
@@ -1259,6 +1259,92 @@ def test_misc_data(self):
             assert(np.all(c.data[-1] == [4., 4., 6., 9., 8.]))
 
 
+class TestDataGather(object):
+
+    @pytest.mark.parallel(mode=4)
+    @pytest.mark.parametrize('rank', [0, 1, 2, 3])
+    def test_simple_gather(self, rank):
+        """ Fill me in."""
```

Comment:
leftover?

Reply:
Thanks. the 3 instances of it will be fixed on next push.